README.txt

```diff
@@ -439,7 +439,7 @@ Solaris 11:
 
 Debian/Ubuntu:
 
-  # apt-get install sun-java6-javadb
+  # apt-get install sun-java8-javadb
 
```

Comment:
Since there is no sun-java8-javadb package in any Debian/Ubuntu version, and sun-java6-javadb is so old it's not even in Wheezy, I'd say pointing to the one in Jessie is the best option. And it's available in Ubuntu 14.04, which is already released.